netlistsvg/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: netlistsvg
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/nturley/netlistsvg.git
+  git_rev: master
+
+build:
+  # number: 201803050325
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20180305_0325
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - TRAVIS
+
+requirements:
+  build:
+    - nodejs
+  host:
+    - nodejs
+  run:
+    - nodejs
+
+test:
+  requires:
+    - symbiflow-yosys
```

Comment:
There's no testing command or script so I don't think this should be here.

BTW. Please add at least some `--version` or `--help` test command. Even that simple command can catch some packaging problems.

Reply:
There's no --help or --version in this program. I've added a simple JSON for testing purpose. Taken from official readme: https://github.com/nturley/netlistsvg#examples